portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase

```diff
@@ -440,8 +440,6 @@ definition {
 
 		SelectFrameTop();
 
-		DataEngine.waitForAutoSave();
-
 		Button.clickPublish();
 
```

Comment:
The use of the macro `DataEngine.waitForAutoSave();` is not necessary.
```
macro waitForAutoSave {
		if (IsElementPresent(key_text = "Publish", locator1 = "Button#ANY")) {
			Button.clickPublish();
		}
```
In the macro, If `Publish` button is present, **Click** and right after, the macro `Button.clickPublish();` is called but does not find the button that was already clicked in the first macro.